Makefile

```diff
@@ -160,6 +160,12 @@ test/functional:
 	# Run the functional tests against an existing cluster. Make sure you have logged in to the cluster.
 	go clean -testcache && go test -v ./test/functional -timeout=80m
 
+.PHONY: test/products
+test/products:
+	# Running the products tests against an existing cluster. Make sure you have logged in to the cluster.
+	$(CONTAINER_ENGINE) pull quay.io/integreatly/delorean-cli:master
+	$(CONTAINER_ENGINE) run -e KUBECONFIG=/kube.config -v "${HOME}/.kube/config":/kube.config:z -v $(shell pwd)/test-containers.yaml:/test-containers.yaml quay.io/integreatly/delorean-cli:master delorean tests run --test-config ./test-containers.yaml --output ./test-results/ --namespace products-tests
+
```

Comment:
we should create a tmp dir and attached it to the container so that we can view the results after the command is finished executing.

Reply:
One concern i have about doing this is how it will get executed on Jenkins/Prow? This will only work locally where someone isn't already running in k8s.

Reply:
If we can make this command optionally use the containerised delorean cli or just run the command locally it would probably be a better approach. This would allow it to be used in both CI where delorean cli is made available by some other means and local development where we distribute the tooling in the container.

Reply:
@mikenairn what do you mean by "containerised delorean cli"?

Reply:
@StanleyKaleta This is running the command in this container quay.io/integreatly/delorean-cli:master which is useful to allow developers to run it without requiring installing dependencies other than docker.

This won't work in CI though, so you need to be able to just run the command as if the delorean cli is installed locally for those envs:

```
delorean tests run --test-config ./test-containers.yaml --output ./test-results/ --namespace products-tests
```

Reply:
ok, so for this I'll need to know where delorean cli is installed and then just run delorean command from that dir, right?

Reply:
You can probably assume it's on the PATH, this will be the case for CI. Should be using the container anywhere else.

Reply:
@mikenairn @wei-lee updated first command to include mounted dir and added second command for CI

Reply:
I'm not sure we have to deal with this right now, but we should endeavour to remove the need for 2 separate make targets. It should be possible to have the command we want to run, run inside the container if we aren't already running in a container, and using the binary directly otherwise.